tests/redirectToRouteTest.php

```diff
@@ -4,10 +4,44 @@
 
 use function Folded\addGetRoute;
 use function Folded\redirectToRoute;
+use Folded\Exceptions\RouteNotFoundException;
 
 it("should return null when using the redirectToRoute function", function (): void {
     addGetRoute("/", function (): void {
     }, "home.index");
 
     expect(redirectToRoute("home.index"))->toBeNull();
 });
+
+it("should set proper location header", function(): void {
+    addGetRoute("/", function (): void {
+    }, "home.index");
+    redirectToRoute("home.index");
+    $headers = xdebug_get_headers();
+    $expectedHeaders = [
+        'Location:/'
+    ];
+    expect($headers)->toEqual($expectedHeaders);
+});
+
+it("should add route params", function (): void {
+
+    addGetRoute("/", function (): void {
```

Comment:
The purpose of your test suggests to have a route `home.index` with url `/user/{user}/test/{test}`, for the method to correctly use `Router::replaceRouteParameters()` correctly

If you use `addGetRoute("/user/{user}/test/{test}` this will do the trick :)